### DISCUSSION TITLE
Daily 19/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#60] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer avancé poco con el material de la plataforma. Estuve revisando el diagrama de casos de uso y encontre algunas cosas para modificar. 
Hoy no pude hacer mucho. Voy a subir mas tarde el diagrama de clases a la carpeta de documentación. Por la tarde voya a participar de la clase sincrónica. Voy a tratar de armar la maqueta del proyecto en Android para subirla al repositorio.

**Comment by BraianTroncoso** [comment]
Ayer pude tocar poco y nada con respecto al proyecto. Estuve con otros trabajos que no me permitieron darle prioridad. Hoy voy a intentar a retomar lo que dejé el domingo.
Si puedo conectar y mostrar user de la db avanzo con las demas funcionalidades.

**Comment by MarielaGomez** [comment]
Ayer avancé con los cuestionarios de desarrollo móvil, todavía queda continuar con los siguientes, se dificulta llevar al día el aprendizaje y absorber los conocimientos
